src/Catty/ViewController/CatrobatTableViewController/CatrobatTableViewController.m

```diff
@@ -179,6 +179,8 @@ - (void)addProjectAndSegueToItActionForProjectWithName:(NSString*)projectName
     self.defaultProject = [Project defaultProjectWithName:projectName projectID:nil];
     if ([self shouldPerformSegueWithIdentifier:segueToNewProjectIdentifier sender:self]) {
         [self hideLoadingView];
+        self.defaultProject.header.landscapeMode = [Util askUserForLandscapeMode];
```

Comment:
Maybe comment this line out and add a ToDo to use it when the askUserForLandscapeMode function is implemented?